test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTestBase.cs

```diff
@@ -28,9 +28,8 @@ public IncrementalTestBase(string testProjectsRoot, string mainProject, string e
             // create unique directories in the 'temp' folder
             var root = Temp.CreateDirectory();
 
-            // recursively copy projects to the temp dir and restore them
+            // recursively already restored copy projects to the temp dir
```

Comment:
No need to have the conversation, I completely agree :smile: 

Comments can be helpful but IMO generally mask code issues where semantics are not obvious enough.